platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java

```diff
@@ -185,6 +272,20 @@ private void respondForbidden(HttpServletResponse httpResponse, String msg) {
     }
   }
 
+  public void setWhiteListContexts(List<String> whiteListContexts) {
+    if (whiteListContexts == null) {
+      return;
+    }
+
+    SYSTEM_PROTECTION_WHITELIST.clear();
+    for (String entry : whiteListContexts) {
+      String[] split = entry.split("=");
```

Comment:
You can also compile a regex for splits, but this doesn't happen during the course of a call through the system, so I don't really care about this one. Just thought I'd mention it.